vocs.config.ts

```diff
@@ -8,13 +8,8 @@ export default defineConfig({
       link: "/getting-started",
     },
     {
-      text: "Timelines",
```

Comment:
Why did you remove the timelines section?
- before: timeline section with `items`
- after: just a link